snyk/client.py

```diff
@@ -82,14 +89,28 @@ def request(
             raise SnykHTTPError(resp)
         return resp
 
-    def post(self, path: str, body: Any, headers: dict = {}) -> requests.Response:
-        url = f"{self.api_url}/{path}"
+    def post(
+        self,
+        path: str,
+        body: Any,
```

Comment:
If this is a `dict`, so can use `Dict`.  If it's either a `dict` or a `str` you can do something like `Union[Dict, str]`.